sdk/tests/conformance2/textures/misc/00_test_list.txt

```diff
@@ -1,4 +1,5 @@
 --min-version 2.0.1 active-3d-texture-bug.html
+canvas-remains-unchanged-after-used-in-webgl-texture.html
```

Comment:
Note: since this was added after the 2.0.0 snapshot was taken, for bookkeeping purposes it should say --min-version 2.0.1. Not a big deal.